lesson25/src/js/validation.js

```diff
@@ -0,0 +1,45 @@
+export const showErrorMessage = (target,message) => {
+    target.classList.add("border-2","border-red-500");
+
+    const errorMessage = document.createElement("p");
+    errorMessage.classList.add("field-invalid","text-sm","my-2","text-rose-600");
+	errorMessage.textContent = message;
+
+	if(!target.nextElementSibling){
+		target.insertAdjacentElement("afterend",errorMessage);
+	}
+}
+
+export const removeErrorMessage = (target) => {
+    target.classList.remove("border-red-500");
+
+    const error = target.nextElementSibling;
+    if(error && error.classList.contains("field-invalid")){
+        error.remove();
+    }
+}
+
+export const checkRequired = (input) => {
+   if(input.value.trim() === ""){
+        showErrorMessage(input,"入力してください");
+   }
+}
+
+export const checkLength = (labelName,maxLength,input) => {
+    if(input.value.length > maxLength){
+        showErrorMessage(input,`${labelName}}は、${maxLength}文字以内にしてください`)
```

Comment:
typo
```suggestion
        showErrorMessage(input,`${labelName}は、${maxLength}文字以内にしてください`)
```

Reply:
Thank you for your comment!
I fixed.
https://github.com/nakagawamai/js-lesson/pull/27/commits/a77d123ca89bab7f790dc53f8db5cf90fa7c10cd